src/PhpSpreadsheet/Calculation/TextData/Extract.php

```diff
@@ -72,6 +72,6 @@ public static function right($value = '', $chars = 1): string
             $value = ($value) ? Calculation::getTRUE() : Calculation::getFALSE();
         }
 
-        return mb_substr($value, mb_strlen($value, 'UTF-8') - $chars, $chars, 'UTF-8');
+        return mb_substr($value ?? '', mb_strlen($value ?? '', 'UTF-8') - $chars, $chars, 'UTF-8');
```

Comment:
The null coalescence should probably be done before this point to avoid using it twice in this line.... for scrutinizer, each null coalescence increases the path count

Reply:
Are you certain about Scrutinizer's behavior here? I believe that PhpUnit's code coverage component treats `??` (but not `?:`) as any other operator, and does not add to the complexity rating when it is used. Does Scrutinizer behave differently?

I agree that we can avoid doing it twice in the same line. For this PR, I wanted to make all the changes exactly the same for easy review. In responding to your next comment, I am proposing to look at all of TextData. Can I fix this minor inefficiency at that time?

Reply:
I could be wrong about Scrutinizer; but technically it's the equivalent of an if/else statement with two paths through the code.

Happy to accept it as is now, with a look at simplifying it later